apps/trading/components/market-banner/market-termination-banner.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import * as Types from '@vegaprotocol/types';
+import type { MockedResponse } from '@apollo/client/testing';
+import { MockedProvider } from '@apollo/client/testing';
+import type { MarketViewProposalsQuery } from '@vegaprotocol/proposals';
+import { MarketViewProposalsDocument } from '@vegaprotocol/proposals';
+import type { Market } from '@vegaprotocol/markets';
+import { MarketTerminationBanner } from './market-termination-banner';
+
+const marketMock = {
+  id: 'market-1',
+  decimalPlaces: 3,
+  tradableInstrument: {
+    instrument: {
+      product: {
+        __typename: 'Future',
```

Comment:
Market type actually doesn't matter here, so I guess it isn't necessary, since we have a single method for getting asset name from each product type...

Reply:
a bunch of specs has been added - 6002-MDET-403